packages/SymbioticController/src/Adapter/Nette/Application/PresenterHelper.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\SymbioticController\Adapter\Nette\Application;
+
+use Nette\Application\UI\Presenter;
+
+final class PresenterHelper extends Presenter
+{
+    public function __construct()
+    {
+        parent::__construct();
+        $this->saveGlobalState();
```

Comment:
It is because of Nette's way of implementation. 

```php
/** @var array */
private $globalParams;
```

Is actually not array unless you set it to - it's default value is `NULL`.
This is causing warnings when trying to use `{link}` or `n:href` in templates.